apps/ledger-live-desktop/src/renderer/components/ReadOnlyAddressField.tsx

```diff
@@ -110,7 +110,7 @@ function ReadOnlyAddressField({ address, allowCopy = true }: Props) {
         </ClipboardSuspicious>
       ) : null}
       <Box horizontal alignItems="stretch">
-        <Address allowCopy={allowCopy}>
+        <Address allowCopy={allowCopy} data-test-id="address-field">
```

Comment:
is it possible to locate this element by text or something else to avoid adding test-id? Same question for all other places where you have added new test-id in this PR

Reply:
If locator test is modified, the test will fail, that why I added some unique ID

Reply:
maybe my point was not clear enough, so let me explain - I mean that we need to avoid adding data-test-id, to do so we must leverage other way to locate elements on the screen - by text, by role, etc etc